apps/remix-ide/src/remixAppManager.js

```diff
@@ -111,6 +111,17 @@ export class RemixAppManager extends PluginManager {
     try {
       const res = await fetch(this.pluginsDirectory)
       plugins = await res.json()
+      plugins.push({
```

Comment:
it is a native plugin, would it be possible to  register and activate it the same way we do for native plugin (see app.js for some code examples)

Reply:
And maybe you'd like to add it to `requiredPlugins` of remixAppManager.js cause it shouldn't be seen from the plugin manager (or should it?)

Reply:
This was just to make it easy to test. No reason to make it native. It doesn’t actually do a lot. It’s the dgitprovider which is native and does all the work

Reply:
it's gone